zqd/handlers_zeek_test.go

```diff
@@ -11,88 +12,142 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/brimsec/zq/pkg/nano"
 	"github.com/brimsec/zq/pkg/test"
 	"github.com/brimsec/zq/zqd"
 	"github.com/brimsec/zq/zqd/api"
 	"github.com/brimsec/zq/zqd/packet"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/stretchr/testify/suite"
 )
 
-func TestPacketPost(t *testing.T) {
-	suite.Run(t, new(PacketPostSuite))
-}
+var zeekpath = os.Getenv("ZEEK")
 
-type PacketPostSuite struct {
-	suite.Suite
-	core     *zqd.Core
-	space    string
-	pcapfile string
-	payloads []interface{}
+type packetPostTest struct {
+	Name           string
+	ZeekExec       string
+	Pcapfile       string
+	ExpectedStatus int
+	Test           func(packetPostResult)
 }
 
-func (s *PacketPostSuite) TestCount() {
-	expected := `
+func TestZeekPacketPost(t *testing.T) {
+	var cases = []packetPostTest{
```

Comment:
I like the breakout of different tests here. I think this array form of test cases is better suited for table-driven tests ( https://github.com/golang/go/wiki/TableDrivenTests ), where the test logic is the same for each case. So, consider if these tests were structured like:
```go
t.Run("SearchCount", func(t *testing.T) {
  pt := setupPacketPostTest(t)
  defer cleanupPacketPostTest(pt)
  ...
})
```

Reply:
Thanks for the post to the wiki! I agree, will change.